src/vue/components/inflections-browser.vue

```diff
@@ -551,6 +552,15 @@ export default {
     if (typeof this.$el.querySelector === 'function') {
       this.htmlElements.content = this.$el
     }
+
+    this.$options.lexrqStartedUnwatch = this.$store.watch((state) => state.app.lexicalRequest.startTime, () => {
```

Comment:
I'm not sure if we need this functionality or not, but it's to collapse all expanded inflection tables in the inflection browser when a new lexical request is started. Because we set a watcher on a Vuex store prop within a Vue instance (not on the Vue instance prop) it won't be destroyed with the Vue instance. This can be the case when we use `v-if` on the UI component itself, I believe. So we store a function to remove a watcher and call it in `beforeDestroy()`.

Reply:
hmm. I don't really know if we need that or not. But if it's retaining existing behavior, let's just keep it for now.